core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitManager.java

```diff
@@ -145,6 +145,16 @@ public DistributedTransaction begin(String txId) {
     return begin(txId, config.getIsolation());
   }
 
+  @Override
+  public DistributedTransaction beginReadOnly() {
+    throw new UnsupportedOperationException("implement later");
+  }
+
+  @Override
+  public DistributedTransaction beginReadOnly(String txId) {
+    throw new UnsupportedOperationException("implement later");
+  }
```

Comment:
I will add an implementation for Consensus Commit in a separate PR after this PR is merged.